diagnose_customer_currency.py

```
@@ -0,0 +1,143 @@
+#!/usr/bin/env python3
```

Ruff (0.14.2)

Shebang is present but file is not executable

(EXE001)